flutter/lib/models/file_model.dart

```diff
@@ -423,6 +429,14 @@ class FileController {
         }
       } else if (options.value.home.isEmpty) {
         options.value.home = fd.path;
+
+        final homeDir = ( await bind.sessionGetPeerOption(
+          sessionId: sessionId, name: "remote_home_dir"));
+
+        if (homeDir.isEmpty){
+          bind.sessionPeerOption(
+            sessionId: sessionId, name: "remote_home_dir", value: fd.path);
+        }
```

Comment:
Do we really need store it as peer option? it will write disk and pollute peer options.

> I could not find any direct implementation for the remote home directory yet but since the home path is correct in the first connection we can just store it according to the session ID and fetch it whenever needed.

Reply:
Ok, I will try to find a better way to get home directory.